docs/guides/webhooks.mdx

```diff
@@ -194,21 +193,98 @@ Once verified, the following is an example of the webhook event body.
       }
     },
     "type": "webhook-event"
-  }
+  },
+  "included": [
+    {
+      "id": "[SCENE_ID]",
+      "type": "scene",
+      "attributes": {
+        "metadata": {
+          "key": "value"
+        }
+      },
+      "links": {
+        "self": { 
+          "href": "scenes/[SCENE_ID]"
+        },
+      }
+    }
+  ],
 }
 ```
 
-There isn't much information in the event for security purposes. To retrieve additional information on the resource, make a `GET` call to the Vertex API using the URL provided in the `data.resource.links.related` field.
+The information in the event is kept to a minimum for security purposes. To retrieve additional information on the resource,
+make a `GET` call to the Vertex API using the URL provided in the `data.resource.links.related` field. 
+
+In the case of CRUD events,
+more information about the resource attributes or associated resources will be present in the `included` resource array. For Update
```

Comment:
Does it start a new line after the comma? That may be unnecessary.

Reply:
It doesn't, but this formatting is a bit weird. I'll update it!